src/pages/home.js

```diff
@@ -25,11 +25,12 @@ class home extends Component {
       bookData: { loading, books },
     } = this.props;
     const popularBook = books.map((book, index) => ({
-      title: formatStringThumbnail(book.title),
+      title: book.title,
       author: book.userHandle,
       bookImage: book.bookImageUrl,
       desc: book.desc,
       userImage: book.userImage,
+      updatedAt: book.createdAt,
```

Comment:
this is a side comment but if you want to use updateAt, maybe it's best to change the code in the backend to match? otherwise might be confusing!!

Reply:
I will fix it to match backend.

Reply:
I changed into createdAt.